awswrangler/mysql.py

```diff
@@ -139,6 +139,7 @@ def connect(
         read_timeout=read_timeout,
         write_timeout=write_timeout,
         connect_timeout=connect_timeout,
+        ssl=attrs.ssl_context,
```

Comment:
Minor - I would put this line right after the `host=attrs.host` (line 137) just to keep all attributes arguments together